pom.xml

```diff
@@ -3,20 +3,20 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>spark-connector-common-java</artifactId>
-    <version>1.0.1</version>
+    <version>1.0.2</version>
```

Comment:
Actually this is a breaking change as it replaces the JDBC URL with host, port and fingerprint. You might want to increment the major version.

Reply:
Great! Yes, updated it now